tests/integration/coordinator/plugins/clickhouse/test_plugin.py

```diff
@@ -313,6 +313,10 @@ async def setup_cluster_content(clients: Sequence[HttpClickHouseClient], clickho
     await clients[0].execute(b"INSERT INTO default.in_object_storage VALUES (123, 'foo')")
     await clients[1].execute(b"INSERT INTO default.in_object_storage VALUES (456, 'bar')")
     await clients[2].execute(b"INSERT INTO default.in_object_storage VALUES (789, 'baz')")
+    # these should be routed to local disk
```

Comment:
Why would they be routed to the local disk?
Even if there is a reason, it is better to test. (check if `system.parts` has parts on all disks for example)

Reply:
I don't think this is actually needed to reproduce the bugs I was seeing.  Removed.

Reply:
I do not understand what is happening here now.
Commit says: "make sure object storage test table has data both on local and remote disk", but there are 0 new tests in this pr, and everything looks to work just as it was. Why are you now sure that the data is stored on both disks during tests?